build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -12,10 +12,39 @@
     <!-- xUnit.net WPF Runner -->
     <XUnitWpfPath>$(NuGetPackageRoot)\xunit.runner.wpf\$(xunitrunnerwpfVersion)\tools\xunit.runner.wpf.exe</XUnitWpfPath>
     <XUnitWpfArguments>"$(OutDir)\$(AssemblyName).dll"</XUnitWpfArguments>
-  </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="xunit.runner.wpf" Version="$(xunitrunnerwpfVersion)" PrivateAssets="all" />
```

Comment:
Does this mean we acquire xunit.runner.wpf any longer? Is there something that needs to happen to bring that back.